src/main/java/club/sk1er/hytilities/handlers/lobby/limbo/LimboLimiter.java

```diff
@@ -33,9 +34,9 @@ public void onTick(TickEvent.ClientTickEvent event) {
     @SubscribeEvent
     public void onWorldChange(WorldEvent.Unload event) {
         limboStatus = false;
+        ChatRestyler.reset(); // putting this here so we don't have to make a new event class just to do this
     }
 
-    @SuppressWarnings("unused")
```

Comment:
its unused in the sense that it's not directly called but is called inside of a transformer, so keep this